spec/datadog/core/telemetry/emitter_spec.rb

```diff
@@ -74,6 +74,17 @@
         end
       end
     end
+
+    context 'with data' do
+      subject(:request) { emitter.request(request_type, data: data) }
+      let(:data) { { changes: ['test-data'] } }
+      let(:request_type) { 'app-client-configuration-change' }
+
+      it 'sends data to the emitter' do
```

Comment:
The test description mentions emitter, but the asserting against `http_transport`.
Is that correct?

Reply:
Renamed to match!